### PR TITLE
[13.0][IMP]product_attribute_value_menu: Products related

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
@ForgeFlow @JordiMForgeFlow 
This PR adds a button on the Attribute Values tree view, which redirects the user to the products in which the value is used.

### CODE REVIEW

**Review by albariera — APPROVED** [review]
LGTM :+1:

**Review by JoanMForgeFlow — APPROVED** [review]
Good idea! seems OK!

**Comment by GuillemCForgeFlow** [comment]
@rousseldenis could you also check this? Thanks! :smile:

File: product_attribute_value_menu/models/product_attribute_value.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    _inherit = "product.attribute.value"
+
+    product_count = fields.Integer(string="Product", compute="_compute_product_count")
+
+    def _compute_product_count(self):
+        for value in self:
+            value.product_count = len(value.pav_attribute_line_ids)
+
+    def action_view_product(self):
+        action = self.env.ref("product.product_template_action").read()[0]
```
Comment (rousseldenis) [inline product_attribute_value_menu/models/product_attribute_value.py]: Use for_xml_id function instead.

```suggestion
        action = self.env['ir.actions.act_window'].for_xml_id("product.product_template_action")
```

File: product_attribute_value_menu/models/product_attribute_value.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                (self.env.ref("product.product_template_only_form_view").id, "form")
+            ]
+            if "views" in action:
+                action["views"] = form_view + [
+                    (state, view) for state, view in action["views"] if view != "form"
+                ]
+            else:
+                action["views"] = form_view
+            action["res_id"] = products.id
+        action["context"] = self._context
```
Comment (rousseldenis) [inline product_attribute_value_menu/models/product_attribute_value.py]:
```suggestion
        action["context"] = self.env.context
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

**Comment by GuillemCForgeFlow** [comment]
> Code review

Thanks! :+1: Could you merge it? @rousseldenis

**Review by JordiMForgeFlow — APPROVED** [review]
Functional review 👍🏼

**Comment by GuillemCForgeFlow** [comment]
This is ready to merge :+1:  @rousseldenis

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@GuillemCForgeFlow minor or major a preference?

**Comment by GuillemCForgeFlow** [comment]
@rousseldenis Can wait, but as soon as you can. Thanks!

**Comment by rousseldenis** [comment]
> @rousseldenis Can wait, but as soon as you can. Thanks!

No, wanted to know if you consider this as major change or minor (I think this is not a breaking change, so minor should be ok)

**Comment by GuillemCForgeFlow** [comment]
@rousseldenis Oh, sorry! First time i heard this! :smiley: Yes, **minor** is fine!

**Pull request merged**